src/api/Alert/types.ts

```diff
@@ -302,3 +303,8 @@ export interface ICasePreview extends Omit<IFullCase, 'statusChangedOn' | 'statu
 export interface IGetAllAlertUsersParams extends IDataSourceParams {
   mutualAlertIds?: number[];
 }
+
+export interface IBulkAssignCasesParams extends IDataSourceParams {
+  casesPerAlert: GenericObject<number[]>;
```

Comment:
I feel `GenericObject<number[]>;` is not explicit: it's not possible to know with confidence what are the keys and the values by just looking at this piece of code. A more explicit alternative could be:

```typescript
  casesPerAlert: {alertId: number; caseIds: number[]}[];
```

with possibly a type alias or an interface for these alertId/caseIds entries.

One benefit I also see is that using such more strongly typed alternative simplifies server-side validation (and possibly also the whole handling of the request).